parm/soca/berror/saber_blocks.yaml

```diff
@@ -1,101 +1,42 @@
-covariance model: hybrid
-components:
-- covariance:
-    # This will setup B = K D C_v C_h C_h^{T} C_v^{T} D K^{T}
-    covariance model: SABER
-    saber central block:
-      saber block name: EXPLICIT_DIFFUSION
-      active variables: [tocn, socn, ssh, cicen]
-      geometry:
-        mom6_input_nml: mom_input.nml
-        fields metadata: ./fields_metadata.yaml
-      group mapping:
-      - name: ocean
-        variables: [tocn, socn, ssh]
-      - name: ice
-        variables: [cicen]
-      read:
-        groups:
-        - name: ocean
-          horizontal:
-            filename: hz_ocean.nc
-          vertical:
-            filename: vt_ocean.nc
-        - name: ice
-          horizontal:
-            filename: hz_ice.nc
+covariance model: SABER
+saber central block:
+  saber block name: EXPLICIT_DIFFUSION
+  active variables: [tocn, socn, ssh, cicen]
+  geometry:
+    mom6_input_nml: mom_input.nml
+    fields metadata: ./fields_metadata.yaml
+  group mapping:
+  - name: ocean
+    variables:
+    - tocn
+    - socn
+    - ssh
+  - name: ice
+    variables:
+    - cicen
+  read:
+    groups:
+    - name: ocean
+      horizontal:
+        filename: hz_ocean.nc
+      vertical:
+        filename: vt_ocean.nc
+    - name: ice
+      horizontal:
+        filename: hz_ice.nc
 
-    linear variable change:
-      input variables: [cicen, hicen, hsnon, socn, tocn, uocn, vocn, ssh]
-      output variables: [cicen, hicen, hsnon, socn, tocn, uocn, vocn, ssh]
-      linear variable changes:
+saber outer blocks:
+- saber block name: StdDev
+  read:
+    model file:
+      date: '{{ATM_WINDOW_MIDDLE}}'
```

Comment:
Should this line and the others in this file have spaces around the variable, inside the curly brackets?